### ISSUE TITLE
Breadcrumb link colors are not mapped to user defined link color in Windows High Contrast Mode

### ISSUE THREAD

**Issue by jlukosky** [OfficeDev/office-ui-fabric-react#10795]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Using Edge and Windows High Contrast mode (black theme) the hyperlink color I have defined is yellow. In the breadcrumb controls the link colors are all gray.

### Please provide a reproduction of the issue in a codepen:

Observe on the [Fabric Breadcrumb control page](https://developer.microsoft.com/en-us/fabric#/controls/web/breadcrumb)

#### Actual behavior:

Browsing to the [Fabric Breadcrumb control page](https://developer.microsoft.com/en-us/fabric#/controls/web/breadcrumb) the link colors of the breadcrumbs is gray instead of the yellow I have defined in my high contrast theme.

#### Expected behavior:
The color of links, including the breadcrumbs, should map to the hyperlink color I've defined in my Windows High Contrast mode theme. The default color for the High Contrast Black theme is bright yellow.

### Documentation describing expected behavior

- [Windows High Contrast Web Development Techniques](https://microsoft.sharepoint.com/teams/msenable/Pages/WindowsHighContrastWebDevelopment.aspx)
- [How to use -ms-high-contrast](http://www.gwhitworth.com/blog/2017/04/how-to-use-ms-high-contrast) by Greg Whitworth
- [Use high contrast mode: Applies to: Windows 10, Windows 8.1, Windows 7](https://support.microsoft.com/en-us/help/13862/windows-use-high-contrast-mode)

**Comment by khmakoto** [comment]
Hi @jlukosky, I cannot seem to repro this neither in my local build, nor in the website.

The website:
![image](https://user-images.githubusercontent.com/7798177/66772813-6f26b180-ee72-11e9-8f41-43394ccc4145.png)

My configuration:
![image](https://user-images.githubusercontent.com/7798177/66772827-78b01980-ee72-11e9-8357-0631511d7b05.png)

As you can see, hyperlinks in `Breadcrumbs` are showing as yellow just as my configuration states. What version of Fabric are you using? That could help us get to the root of the problem.

**Comment by ecraig12345** [comment]
There's currently a known issue which is interfering with this scenario--but even once that's fixed, the behavior may not be exactly what you're expecting.

Some background: A breadcrumb item's action can be defined using `onClick` and/or `href`. Most of the example items happen to use only `onClick`. For a long time the intended behavior was that if an item had a `href`, it would be rendered as a link; otherwise it would be rendered as a button. (Items without either `onClick` or `href` would be rendered as spans.)

PR #10113 changed it so *all* items render as links. However, this breaks "clicking" `href`-less items by pressing enter (even if `onClick` is defined, enter isn't interpreted as a click). So I'm reverting that change as part of #10736.

Using [this link](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/10736/merge/fabric-website-resources/dist/index.html#/examples/breadcrumb) you can see what it will look like after the revert: items with `href`s are links (link color), items with only `onClick` are buttons (button color), and non-actionable items are spans (text color).

**Comment by jlukosky** [comment]
My expectation is a link or link-type construct that navigates to a new destination (either on a new page or somewhere on the current page) will function and appear as a link. A button function (which shouldn't be used for navigation, btw) would show as a button. 

On the page you link to I noticed a great number of the "links" are implemented as buttons. That role is incorrect. Ideally these would be coded as semantic links using the `a` element. If not, please add role="link" where appropriate.

**Comment by ecraig12345** [comment]
We don't have control over how people choose to use breadcrumbs, so it's possible in some scenarios that the breadcrumb will do some kind of in-page navigation (handled by onClick) rather than anything that changes the URL. In that case, using `a` for the breadcrumbs would prevent them from being "clickable" with the keyboard since they have no `href`, which is pretty terrible for accessibility too. So I'm not sure there's a great choice for how to render them besides as buttons.

Re: tab focus, each breadcrumb group is contained in a focus zone, which means there's a single tab stop and the arrow keys are used to navigate between items.

**Comment by jlukosky** [comment]
> Breadcrumbs should be used as a navigational aid in your app or site 

It seems the default should be anchors, and they should be coded that way. Using buttons is the wrong pattern. _BUT ANYWAY...._

This bug regards them not reflecting the hyperlink color so if that's done then this issue is done. Other things can be handled separately.

**Comment by micahgodbolt** [comment]
Not all websites do navigation via links and urls. I took a quick look at OneDrive and they use breadcrumbs as buttons and use a query string to change their view:

```
...onedrive.aspx?id=%2Fpersonal%2Fmgodbolt%5Fmicrosoft%5Fcom%2FDocuments%2FPresentations%2Fdev%20kitchen```

So our practice is:

Use links: `{ text: 'Files', key: 'Files', href: '#/controls/web/breadcrumb' }` and get an anchor tag

Use onClicks:  `{ text: 'Files', key: 'Files', onClick: _onBreadcrumbItemClicked },` and get a button. 


So the question is, "is it correct to treat buttons and links visually the same in the breadcrumb".

That's a question I'd defer to design, as we implement their specs. 

@betrue-final-final, does design have a strong opinion on buttons and links looking visually the same in breadcrumbs (especially in high contrast scenarios).

**Comment by jlukosky** [comment]
> Not all websites do navigation via links and urls.

Navigation elements should semantically be links, not buttons. If other sites are using buttons to navigate to another page or a place in the current page then perhaps they're not correct.

Coloring the buttons yellow indicates the intention that these function as links so why would they not use the correct syntax?

**Comment by micahgodbolt** [comment]
Taking a look at this again, it appears that we are doing this as expected in our actual controls, and that the MWF styling are causing problems once again. Here's the raw control without the MWF css

https://fabricweb.z5.web.core.windows.net/oufr/7.67.0/#/examples/breadcrumb

 
![image](https://user-images.githubusercontent.com/1434956/69983166-e4b21400-14ea-11ea-87a7-fab28666e1b8.png)


As you can see, the links use the purple link text and buttons use the white button text

![image](https://user-images.githubusercontent.com/1434956/69983205-fdbac500-14ea-11ea-9ebd-5d690e00e4bd.png)
